RoadTripMoustache/kana_to_kanji
# perf(glossary): Improve `KanaList` render performance

## 📖 Description


Improve the render time of the `KanaList` by switching `GridView.count` to `builder`.

### 🧪 How to test the change?




1. Load the glossary tab
2. See changes

### ☑️ Checklist before requesting a review
